### DISCUSSION TITLE
about flashmq and vernemq

### DISCUSSION THREAD

**Discussion by Helloworld152** [halfgaar/FlashMQ#70] [Q&A] [answered]
If I have two servers, one with flashmq and one with vernemq, can I bridge them to share messages？

**Comment by bigsmoke** [comment, marked as answer]
FlashMQ does indeed support bridge mode since version 1.7. Though I have not tested it myself, I do not see any problems in mixing vendors (except, of course, that VerneMQ will have trouble keepong up with FlashMQ at full capacity 😉).

**Reply by Helloworld152** [reply]
Thanks! I have another question:Can Flashmq start multiple processes at the same time according to different configuration files?Like Redis?

**Reply by halfgaar** [reply]
Yeah, you can start it with `flashmq --config-file example.conf`, provided you have no conflicts in your listeners or file/directory locations. Also, you'll have to create your own systemd service files, or other means of starting it.
